jetson/ra_kinematics/main.cpp

```diff
@@ -111,7 +111,7 @@ int main() {
 
     lcmHandlers handler(&robot_arm);
 
-    lcmObject.subscribe( "/arm_position", &lcmHandlers::armPositionCallback, &handler );
+    lcmObject.subscribe( "/ra_ik_cmd", &lcmHandlers::armPositionCallback, &handler );
```

Comment:
should this be raIKCmdCallback for consistency?

Reply:
I think it's being worked on in ik-integration

Reply:
yea it def is https://github.com/umrover/mrover-workspace/blob/ik-integration/jetson/ra_kinematics/main.cpp#L37

Reply:
perfect